core/getting-started/unit-testing-with-fsharp/MathService.Tests/Tests.fs

```diff
@@ -5,14 +5,6 @@ open Xunit
 
 open MathService
 
-[<Fact>]
```

Comment:
I understand why this existed, but with the NUnit changes it no longer makes sense so I completely removed it @IEvangelist